processing/p5.js-web-editor
# Mobile Sketch Preview Screen

Part of the Mobile UI project. This PR adds a (working!) Mobile Sketch Preview screen (/mobile/preview), which renders the canvas with the project files and resizes it to properly fit the mobile view.

How to test:
1. Run the app locally with the `MOBILE_ENABLED` flag set to `true` (`MOBILE_ENABLED=true npm start`)
  - You should be able to access `/mobile`
  - You should be able to edit code
  - Clicking the play icon should run the sketch and you to the preview screen screen
  - You should be able to go back to the code screen, and repeat the process without errors
  - Clicking the exit icon should take you to the default editor
2. Run the app without the flag enabled
  - It should work as usual
  - `/mobile` routes should not be accessible



I have verified that this pull request:

* [x] is from a uniquely-named feature branch and has been rebased on top of the latest master. (If I was asked to make more changes, I have made sure to rebase onto master then too)
